pulseguardian/guardian.py

```diff
@@ -57,8 +58,31 @@ def clear_deleted_queues(self, queues):
         for queue in deleted_queues:
             logging.info("Queue '{0}' has been deleted.".format(queue))
             db_session.delete(queue)
+
+        # Cleanup bindings on queues that are not deleted:
+        for queue_name in alive_queues_names:
+            self.clear_deleted_bindings(
+                pulse_management.queue_bindings(config.rabbit_vhost, queue_name),
```

Comment:
Since we have to pass in the queue_name anyway, maybe clear_deleted_bindings() should do the pulse_management query?